go/client/pgp_ui.go

```diff
@@ -73,9 +73,9 @@ func (p PgpUI) OutputSignatureSuccessNonKeybase(ctx context.Context, arg keybase
 	}
 
 	if signedAt.IsZero() {
```

Comment:
There is something wrong with this but I can't figure out what without looking deeper at go-crypto code again.

This is what I was seeing before when testing this:
```
Signature verified. Signed by key B42F0AF18858CC8C (unknown to keybase) a long while ago (1754-08-31 00:07:41.128654848 +0124 LMT).
```

so it's set to some value treated as 0 but not detected by `IsZero()`.

Reply:
this is not an issue anymore though because of the change in `go/engine/pgp_decrypt.go`, which makes it so `OutputSignatureSuccessNonKeybase` is not called anymore.

Reply:
I think this is what's happening with the https://play.golang.org/p/tSF-Kj6fPM2

Reply:
Ugh I had a comment here, it ended up in a review for some reason.

Reply:
should I see this comment somewhere? I'm confused

Reply:
I'm going to finish this ticket later today, haven't had that much time for it over the past few days.

Reply:
OK I fixed this bug too